src/views/Profile/components/ClaimGiftModal.tsx

```diff
@@ -22,8 +22,12 @@ export const useCanClaim = () => {
   useEffect(() => {
     const fetchClaimStatus = async () => {
       const claimRefundContract = getClaimRefundContract()
-      const walletCanClaim = await claimRefundContract.canClaim(account)
```

Comment:
This function throws an error on first load on **testnet** (the error is thrown on `develop` testnet too)- 

```
Unhandled Rejection (Error): call revert exception (method="canClaim(address)", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.4.0)
```

The error is handled safely by wrapping it in a try/catch.